README.md

```diff
@@ -34,6 +34,12 @@ _Note:_ These features may not have functional parity with the juju cli at this
 
 1. Clone the repository
 1. Enter the repository directory
+1. Build the provider dependencies using the make `install-dependencies` target:
```

Comment:
As we're not actually building the dependencies, suggest:

"Install the provider requirements and dependencies using the make `install-dependencies` target:"